web/src/components/tasks/business-formation/getErrorStateForField.ts

```diff
@@ -152,12 +152,13 @@ export const getErrorStateForField = (inputParams: {
     const isAvailable = businessNameAvailability?.status === "AVAILABLE";
     const isValid = exists && isAvailable;
     let label = errorState.label;
+
     if (!exists) {
       label = Config.formation.fields.businessName.errorInlineEmpty;
     } else if (businessNameAvailability?.status === undefined) {
       label = Config.formation.fields.businessName.errorInlineNeedsToSearch;
     } else if (businessNameAvailability?.status !== "AVAILABLE") {
-      label = Config.formation.fields.businessName.errorInlineUnavailable;
```

Comment:
[pebble] if we are not using this anymore, we should remove it from the json and yaml config files

Reply:
Yes!